src/main/java/com/thoughtworks/device/SimulatorManager.java

```diff
@@ -65,12 +73,43 @@ public Device getSimulatorDetailsFromUDID(String UDID) throws IOException, Inter
         List<Device> allSimulators = getAllAvailableSimulators();
         Optional<Device> device = allSimulators.stream().filter(d ->
                 UDID.equals(d.getUdid())).findFirst();
-        return device.orElseThrow(()->
+        return device.orElseThrow(() ->
                 new RuntimeException("Device Not found")
         );
     }
 
     @Override
+    public void captureScreenshot(String UDID, String fileName, String fileDestination) throws IOException, InterruptedException {
+        commandPromptUtil.runCommandThruProcess("xcrun simctl io " + UDID + " screenshot "
```

Comment:
Does it propagated from the other PR? if so can we throw the error for xcode less than 8.2.